src/app/home-page/widget/entry-box/entry-box.component.ts

```diff
@@ -82,19 +81,24 @@ export class EntryBoxComponent extends Base implements OnInit {
   }
 
   getMyEntries() {
-    this.userEntries$ = this.usersService.getUserEntries(null, this.filterText, this.entryTypeParam);
-    this.userEntries$
+    this.usersService
+      .getUserEntries(null, this.filterText, this.entryTypeParam, 'response')
       .pipe(
         finalize(() => (this.isLoading = false)),
         debounceTime(750),
         takeUntil(this.ngUnsubscribe)
       )
       .subscribe(
-        (myEntries: Array<EntryUpdateTime>) => {
-          myEntries.forEach(() => {
-            this.listOfEntries = myEntries.slice(0, 7);
-            this.totalEntries = myEntries.length;
-          });
+        (myEntries: HttpResponse<EntryUpdateTime[]>) => {
+          const url = new URL(myEntries.url);
+          if (url.searchParams.get('filter')) {
+            this.listOfResults = myEntries.body;
```

Comment:
If there's lots of filtered results, do we want to `slice` them also, to limit the number?

Reply:
Good idea since we only display a max of 5 results anyway, will just add in a `totalResults` varaible so we can still display the number of entries matched